concent_api/concent_api/settings/base.py

```diff
@@ -331,3 +331,7 @@
 
 # Verifier setting defining number of threads used by Blender
 BLENDER_THREADS = 1
+
+# Settings for creating locks on concent workers ensuring, that only one worker processes particular Subtask in one time
+MAXIMUM_NUMBER_OF_RETRIES = 20
+STARTING_WAIT_TIME_FOR_RETRY_IN_SECONDS = 1
```

Comment:
I think you should delete `_in_seconds` because all ours time settings are in seconds.
Maybe better name for this settings would be something like `TIME_RETRY_INTERVAL` or something like this. 
And there should be comment to this settings also what is it for.

Reply:
done